fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -93,6 +93,7 @@ public enum ErrorCode {
             "View's SELECT and view's field list have different column counts"),
     ERR_NO_DEFAULT_FOR_FIELD(1364, new byte[] {'H', 'Y', '0', '0', '0'},
             "Field '%s' is not null but doesn't have a default value"),
+    ERR_NO_SUCH_QUERY(1365, new byte[] {'H', 'Y', '0', '0', '0'}, "Unknown query id: %s"),
```

Comment:
Use 42000

Reply:
ok. changed in this pr https://github.com/StarRocks/starrocks/pull/48114